src/components/colors-bar/tests/store.js

```diff
@@ -0,0 +1,71 @@
+const store = {
```

Comment:
Please use correct file naming as we use on our project.
For this file, it should look like tests/colors-bar.variables.js